### PR TITLE
Vet Clinic - Database Creation and Animals table creation with queries

### PR DESCRIPTION
**Pull request by jmontanholi**
# In this feature branch I implemented the following:

* # Database
   * ## Database schema - Animals Table with all keys and types
      * ### Inserted information inside the Animals table
      * ### Created queries that select specific information from the Animals table

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Invalid submission 

Hi,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a PR, that has wrong Gitflow. The pull request is supposed to be into development from feature not from development to main. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by jmontanholi** [comment]
Hello @arslanbisharat, first of all, thanks for the message, but Gitflow is supposed to be a feature branch into the main branch, there is no development branch involved. Or am I mistaken?

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```
Comment (Davidosky007) [inline queries.sql]: * Kindly use the proper query as this '%mon%' would look for the string "mon" anywhere in the names of all animals while the question says select only animals that end with "mon". Hint 💭 : this should be the right query '%mon'

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
-    name varchar(100)
-);
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals(
+  id INT PRIMARY KEY, 
+  name CHAR(30), 
+  date_of_birth DATE, 
+  escape_attempts INT, 
+  neutered BOOL,
```
Comment (Davidosky007) [inline schema.sql:10]: * Kindly add varchar for all your columns as it determines the length of characters this is best practice 👍

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @jmontanholi 👋, 

Good job so far! 💪 
* Nice PR ✔️ 
* Descriptive commit messages ✔️ 
* Good looking readme file ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**